ui/src/views/OpenVPNRW.vue

```diff
@@ -22,6 +22,51 @@
 
 <template>
   <div>
+    <!-- pending changes -->
+    <div
+      v-if="pendingChanges"
+      :class="['alert', 'alert-warning', 'alert-dismissable', 'mg-top-10']"
+    >
+      <button
+        class="btn btn-primary pull-right"
+        data-toggle="modal"
+        data-target="#applyPendingChangesModal"
+      >
+        {{$t('openvpn_rw.apply_changes')}}
+      </button>
+      <span :class="['pficon', 'pficon-warning-triangle-o']"></span>
+      <strong>{{$t('warning')}}.</strong>
```

Comment:
It's a matter of taste, but we always use `:` after `Warning. We change it to:
```js
<strong>{{$t('warning')}}:</strong>
```

Reply:
Ok. I actually copied it from the firewall module: https://github.com/NethServer/nethserver-firewall-base/blob/master/ui/src/App.vue#L131